app/javascript/packs/service_active_docs.js

```diff
@@ -9,5 +10,9 @@ document.addEventListener('DOMContentLoaded', () => {
 
   const responseInterceptor = (response) => autocompleteOAS3(response, accountDataUrl)
 
-  ActiveDocsSpec({ url, responseInterceptor }, containerId)
+  window.swaggerUi = new SwaggerUI({
```

Comment:
I think you don't need to export it to window, in the liquid template it is called directly:
```suggestion
  SwaggerUI({
```